src/platform/site-wide/announcements/actions/index.js

```diff
@@ -6,10 +6,12 @@ export const DISMISS_ANNOUNCEMENT = 'DISMISS_ANNOUNCEMENT';
 export const ANNOUNCEMENTS_LOCAL_STORAGE = 'DISMISSED_ANNOUNCEMENTS';
 
 const previouslyDismissedAnnouncements = (() => {
-  let parsed = [];
+  let parsed = null;
 
   return {
     initializeFromLocalStorage() {
+      parsed = [];
+
```

Comment:
`initializeFromLocalStorage ` is called during during unit tests to blank out the state. If the tests ran in a certain order, `parsed` wouldn't overridden by the value in `localStorage` in the next few lines, so sometimes the result of one unit test would carry to the next.